cms/core/tests/test_blocks.py

```diff
@@ -532,3 +536,43 @@ def test_render_block__ds_component_markup(self):
         </table>
         """
         self.assertInHTML(expected, rendered)
+
+
+class NoticeBlockTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.notice_data = {
+            "when": datetime(2025, 1, 1),
+            "text": "Notice text",
+        }
+
+    def test_render_block(self):
+        block = NoticeBlock()
+        rendered = block.render(self.notice_data)
+
+        self.assertIn("1 January 2025", rendered)
+        self.assertIn("Notice text", rendered)
+
+
+class CorrectionBlockTestCase(TestCase):
+    def setUp(self):
+        self.series = ArticleSeriesPageFactory()
+        self.statistical_article = StatisticalArticlePageFactory(parent=self.series)
```

Comment:
Suggestion: These should be in `setUpTestData` so they're only generated once

Reply:
Moved.